src/main/java/org/group23/Survey.java

```diff
@@ -1,26 +1,51 @@
 package org.group23;
 
+import jakarta.persistence.*;
+
 import java.util.ArrayList;
 import java.util.List;
 
+@Entity
 public class Survey {
 
-    private int id;
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    private String name;
+
+    @OneToMany(cascade = CascadeType.ALL)
     private List<Question> questions;
 
-    public Survey(){
-        this.questions = new ArrayList<Question>();
+    public Survey(){}
+
+    public Survey(String name){
+        this.name = name;
+        this.questions = new ArrayList<>();
     }
 
-    public void addQuestion(Question question){
```

Comment:
I think we should leave in addQuestion and removeQuestion methods - even though they won't be used by the repository, they'll be nice to have for controllers and the rest of the app.

Reply:
I will put them back in. Wasn't sure which method of adding questions was better but might as well keep both for now.